recipe/meta.yaml

```diff
@@ -1,21 +1,22 @@
-{% set version = "0.3.0" %}
+{% set version = "1.0.0.dev" %}
```

Comment:
Should this version be less?  Somewhere near 0.4.0?

Reply:
I just got what is in the archived tarball BTW.

Reply:
I don't think this will work with setting at 1.0.0.dev, as that tag is from some time ago. I think we always want to build the current latest version.

If that is not possible, and for conda-forge there always needs to be a tag to build, we can add some new tags like alpha 1, 2, 3, ... etc each time we want to update this build.

Reply:
[`rasterio`](https://github.com/conda-forge/rasterio-feedstock/blob/dev/recipe/meta.yaml#L1) and [`fiona`](https://github.com/conda-forge/fiona-feedstock/pull/59/files) as using alpha releases so, b/c `geopandas` users are familiar with those libraries, we should do the same here instead of using the version + git hash from versioneer. What do you think?

Reply:
Yes, although they use it for their 'main' release as well, while I think for now we don't want people to automatically install this dev version. 
It's a bit more work to tag each time to update this build, but also not that much work. So either is fine with me.